src/Row.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Tile from './Tile'
+
+
+
+const Row = ({row}: {row: Tile[]}) => {
```

Comment:
Why do we need Row when we have Board? Would it be easier to do everything in Board?

Reply:
IF I KNEW THE ANSWER I WOULDVE DONE IT